packages/rainbowkit/src/components/Avatar/Avatar.tsx

```diff
@@ -45,7 +45,7 @@ export function Avatar({ address, imageUrl, loading, size }: AvatarProps) {
       >
         <AvatarComponent address={address} ensImage={imageUrl} size={size} />
       </Box>
-      {typeof loading === 'boolean' && (
+      {loading && (
```

Comment:
Make sure to only show the spinner if it's `true`. Since `typeof loading === 'boolean'` will always be true because it was always a boolean. It had `false` always.